src/constants/externalResources.ts

```diff
@@ -117,14 +117,14 @@ export const getHoneycombSystemMetricsUrl = (
 ): string => {
   const query = {
     calculations: [
-      { op: "AVG", column: "system.memory.usage.used" },
-      { op: "AVG", column: "system.cpu.utilization" },
-      { op: "RATE_AVG", column: "system.network.io.transmit" },
-      { op: "RATE_AVG", column: "system.network.io.receive" },
+      { column: "system.memory.usage.used", op: "AVG" },
+      { column: "system.cpu.utilization", op: "AVG" },
+      { column: "system.network.io.transmit", op: "RATE_AVG" },
+      { column: "system.network.io.receive", op: "RATE_AVG" },
     ],
-    filters: [{ op: "=", column: "evergreen.task.id", value: taskId }],
-    start_time: getUnixTime(new Date(startTs)),
     end_time: getUnixTime(new Date(endTs)),
+    filters: [{ column: "evergreen.task.id", op: "=", value: taskId }],
+    start_time: getUnixTime(new Date(startTs)),
   };
```

Comment:
@ybrill This changed in this pr and the keys are sorted now. I tried in staging and it appears the links still work I just wanted to call it out just in case.

Reply:
I don't see the change. But I'm with you that the order shouldn't matter.

Reply:
I had to revert it the change broke some other things.